src/main/java/uk/gov/cslearning/catalogue/api/CourseController.java

```diff
@@ -97,30 +88,102 @@ public CourseController(CourseRepository courseRepository, CourseService courseS
         return ResponseEntity.ok(new PageResults<>(results, pageable));
     }
 
-    @PutMapping(path = "/{courseId}")
-    public ResponseEntity<Void> update(@PathVariable("courseId") String courseId, @RequestBody Course course) {
-        LOGGER.debug("Updating course {}", course);
-        if (!courseId.equals(course.getId())) {
-            return ResponseEntity.badRequest().build();
-        }
-        if (!courseRepository.existsById(courseId)) {
-            return ResponseEntity.badRequest().build();
-        }
-        courseRepository.save(course);
+    @GetMapping(params = {"mandatory", "department"})
+    public ResponseEntity<PageResults<Course>> listMandatory(@RequestParam("department") String department,
+                                                             @RequestParam(value = "status", defaultValue = "Published") String status,
+                                                             Pageable pageable) {
+        LOGGER.debug("Listing mandatory courses for department {}", department);
 
-        return ResponseEntity.noContent().build();
+        Page<Course> page = courseRepository.findMandatory(department, status, pageable);
+        return ResponseEntity.ok(new PageResults<>(page, pageable));
+    }
+
+    @RoleMapping("ORGANISATION_AUTHOR")
```

Comment:
As with the SearchController it would be good to test the role mappings and perhaps return a 403 if a suitable role/authority isn't found.